lib/src/android/support/design/backlayer/build.gradle

```diff
@@ -1,6 +1,8 @@
 apply plugin: 'com.android.library'
+apply plugin: 'com.github.dcendents.android-maven'
 
-archivesBaseName = 'design'
+archivesBaseName = 'design-backlayer'
+version = '1.0'
```

Comment:
do we want this version to be consistent? Maybe a similar technique to ${project.rootProject.ext.supportVersion} can be used.